src/components/pages/Profile.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import { StyleSheet, Text, View, Button } from "react-native";
+import { connect, MapDispatchToProps, MapStateToProps } from "react-redux";
+
+import { Video } from "expo";
+import { RahaState } from "../../store";
+import { getMembersByIds } from "../../store/selectors/members";
+import { Member } from "../../store/reducers/members";
+
+type OwnProps = {
+  navigation: any;
+  member: Member;
+  isOwnProfile: boolean;
+};
+
+type StateProps = {};
+
+type DispatchProps = {
+  mint: () => void;
```

Comment:
Remove "mint" code from Profile and leave only in MyProfile

Reply:
Will move both to `actions/`